content/PreparedStatement/PreparedStatement.facts.md

```diff
@@ -0,0 +1,62 @@
+### Facts for Prepared Statement lesson:
+
+A `prepared statement` is used to execute an SQL statement multiple times with the same or different parameter values. The execution of a prepared statement consists of using `prepare()` and `execute()` methods. 
+
+- The `PDO::prepare()` method prepares an SQL statement for execution using the `PDOStatement::execute()` method and returns a `PDOStatement` object.
```

Comment:
Can we simplify this. Let's not include `PDOStatement::execute` in this line. Then on the `PDOStatement::execute` let's say something like `it executes a PDOStatement object.